packages/create-canon/README.md

```diff
@@ -0,0 +1,15 @@
+# @datawheel/create-canon
+
+A quick script to create a new canon app.
+
+## Usage
+
+```bash
+npm init @datawheel/canon
```

Comment:
I'm just learning about this `init` command, wouldn't this be `npm init canon` if our package is called `create-canon`?

Reply:
basically, yes
`npm init something` will look for a package called `create-something` on the registry